lib-multisrc/mangahub/src/eu/kanade/tachiyomi/multisrc/mangahub/MangaHubDto.kt

```diff
@@ -0,0 +1,84 @@
+package eu.kanade.tachiyomi.multisrc.mangahub
+
+import kotlinx.serialization.Serializable
+
+typealias ApiChapterPagesResponse = ApiResponse<ApiChapterData>
+typealias ApiSearchResponse = ApiResponse<ApiSearchObject>
+typealias ApiMangaDetailsResponse = ApiResponse<ApiMangaObject>
+
+// Base classes
+@Serializable
+class ApiResponse<T>(
+    val data: T?,
+    val errors: List<ApiErrorMessages>?,
+)
+
+@Serializable
+class ApiErrorMessages(
+    val message: String,
+)
+
+// Chapter metadata (pages)
+@Serializable
+class ApiChapterData(
+    val chapter: ApiChapter?,
+)
+
+@Serializable
+class ApiChapter(
+    val pages: String,
+)
+
+@Serializable
+class ApiChapterPages(
+    val p: String,
+    val i: List<String>,
```

Comment:
```suggestion
    @SerialName("p") val page: String,
    @SerialName("i") val images: List<String>,
```
meaningful names should be used imo